### ISSUE TITLE
Make service hostnames configurable

### ISSUE THREAD

**Issue by xeals** [AnalogJ/lodestone#81]
I was looking to integrate Lodestone into an existing Docker stack and renamed some of the containers to be clearer to me (e.g., the storage container). For some services this can't happen as service hostnames are hard-coded into the web app and processor:

https://github.com/AnalogJ/lodestone/blob/039fa76e5744b1de6ed69cd200ed6f286aeeee33/web/backend/routes/status.js#L5
https://github.com/AnalogJ/lodestone/blob/7d6d0b591e17e4ab40e83ebbe7fa28eb532745fa/web/backend/routes/sync.js#L162

If the Docker DNS can't find a host named `elasticsearch` or `rabbitmq` it will fail. This means that containers are required to be named as specified in the sample `docker-compose.yml`.

A solution to this is to draw these host names from the environment or some other form of configuration.

If this is unable to be done, it might be worth including somewhere in the documentation somewhere that something like Docker network aliases would be required:

```yaml
lodestone_storage:
  # ...
  networks:
    lodenet:
      aliases:
        - storage
```

**Comment by AnalogJ** [comment]
Hey @xeals This should be fixed by the changes in the beta2 branch (#75)
Can you take a look and confirm?

**Comment by xeals** [comment]
Forgot to check PRs when I was looking to see if someone had reported this. My bad! Thanks for this. Unable to test at the moment without diving into getting the processor Dockerfile to build with the latest PR, but I'll trust that it'll be ready by the time the next beta is out.
